src/components/ha-form/ha-form-grid.ts

```diff
@@ -26,10 +33,19 @@ export class HaFormGrid extends LitElement implements HaFormElement {
 
   @property() public computeHelper?: (schema: HaFormSchema) => string;
 
-  protected firstUpdated() {
+  protected firstUpdated(changedProps: PropertyValues) {
+    super.firstUpdated(changedProps);
     this.setAttribute("own-margin", "");
   }
 
+  protected updated(changedProps: PropertyValues): void {
+    if (changedProps.has("schema")) {
+      if (this.schema.min_width) {
+        this.style.cssText = `--form-grid-min-width: ${this.schema.min_width}`;
```

Comment:
Why not use [`setProperty`](https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleDeclaration/setProperty) ?

Reply:
hm no that's for normal properties, not var